tournament/tournament.rb

```diff
@@ -0,0 +1,13 @@
+class Tournament
+  def self.tally(input)
+    banner + standings(input).to_s
+  end
```

Comment:
`standings(input).to_s` has `.to_s` as a safeguard in case standings returns nil